### DISCUSSION TITLE
Can you detect if an updateOne operation is successful or not?

### DISCUSSION THREAD

**Discussion by Atlinx** [twinlogix/typetta#199] [Q&A] [answered]
Is it possible to detect if an updateOne operation on a DAO is successful? It looks like it only returns a `Promise<void>`.

**Comment by minox86** [comment, marked as answer]
Every Typetta operation return in case of success or throws in case of error. You just need to do something like:
```
try {
   await entityManager.user.updateOne(...)
} catch (e) {
  // handle error
}
```
